visualization/app/codeCharta/codeCharta.component.ts

```diff
@@ -33,7 +35,7 @@ export class CodeChartaController {
 		// @ts-ignore
 		private injectorService: InjectorService // We have to inject it somewhere
 	) {
-		this._viewModel.version = codeCharta.version
+		this._viewModel.version = getVersion()
```

Comment:
I would not change this. The helper does not reduce the code and before it was immediately clear what version was read. Now it's an indirection and the helper function is just extra code that has to be maintained. Was there a specific reason why you changed this?

Reply:
I felt like importing the package.json was a bit ugly and we do that a lot. Basically whenever we needed either the current version of CodeCharta or the current API version.
`codeCharta.version` reads well, however codeCharta is actually the package.json. 

I don't know what a good practice is here. I just though that importing the package.json in like 10 files was ugly.

Reply:
I am not aware of any best practice when it comes to this. It is not that common to read the current modules version. And the cases that did this just accessed the package json directly.

I suggest to use `packageJSON` as variable name instead of `codeCharta`. That way it's even clearer what the version corresponds to.